package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/shopify-express",
-  "version": "1.0.0-alpha.2",
+  "version": "1.0.0-alpha.3",
```

Comment:
version bump just because of the readme?

Reply:
nvm; I'm blind :(

Reply:
Nah it's because I fudged an earlier PR. This commit does need to go in.